### PR TITLE
minor bug fixed on link chat groups

### PR DESCRIPTION
**Pull request by ankitbhardwaj78**

### CODE REVIEW

**Comment by bloggervista** [comment]
Hi thanks but can i ask something. Was there any reason to remove mysql from production gem file?

File: Gemfile [hunk 1]
```diff
@@ -45,9 +45,7 @@ group :test do
   gem 'simplecov', '~> 0.14', require: false
 end
 
-group :production do
-  gem 'mysql2', '~> 0.4.5'
```
Comment (bloggervista) [inline Gemfile]: any reason to remove it?

**Comment by ankitbhardwaj78** [comment]
In my point of view there is no need to add it.....in production mode
postgressql is better to use

**Comment by bloggervista** [comment]
@ankitbhardwaj78  yeah I will also recommend post gres. I guess I will need to open issue for adding support to postgrest. Will merge it as your point is valid

**Pull request merged**